packages/compass-e2e-tests/tests/connection.test.ts

```diff
@@ -247,7 +247,273 @@ async function assertCannotCreateCollection(
 /**
  * Connection tests
  */
-describe('Connection screen', function () {
+describe('Connection string', function () {
+  let compass: Compass;
+  let browser: CompassBrowser;
+
+  before(async function () {
+    compass = await init(this.test?.fullTitle());
+    browser = compass.browser;
+  });
+
+  after(function () {
+    return cleanup(compass);
+  });
+
+  afterEach(async function () {
+    await screenshotIfFailed(compass, this.currentTest);
+    await disconnect(browser); // disconnect AFTER potentially taking a screenshot
+  });
+
+  it('can connect using connection string', async function () {
+    await browser.connectWithConnectionString();
+    if (!TEST_COMPASS_WEB) {
```

Comment:
`await browser.connectWithConnectionString()` already checks that we connected so we don't _really_ need this additional assertion.